backend/api/Makefile

```diff
@@ -22,7 +22,7 @@ PREBUILT_REMOTE_IMAGE=gcr.io/ml-pipeline-test/api-generator@sha256:431635b564a87
 
 # Generate clients using a pre-built api-generator image.
 .PHONY: generate
-generate: hack/generator.sh $(API_VERSION)/*.proto
+generate: hack/generator.sh $(API_VERSION)/api/*.proto
```

Comment:
I might be missing something. Is it really necessary to create this subfolder and regenerate everything for v1beta1?

Reply:
Yes, otherwise the Python package will pick up the folder name as part of the package name. See [here](https://developers.google.com/protocol-buffers/docs/reference/python-generated#package):
```
The Python code generated by the protocol buffer compiler is completely unaffected by the package name defined in the .proto file. Instead, Python packages are identified by directory structure.
```

Reply:
Hi @Linchin!
I might be missing something here too. What Python package did you mean? Is it the one you generate in `backend/api/build_kfp_server_api_python_package.sh`?

Reply:
@Linchin The location of the proto files has no correlation to the location of the generated python files. You can put proto files anywhere, while maintaining the desired python package name. Think about why there's no change to `v1beta1/python_http_client` after you moved the v1beta proto files.

Reply:
@gkcalat Yes, that's what I am talking about.